.github/workflows/build-llvm.yml

```diff
@@ -6,6 +6,27 @@ on:
 name: Build LLVM Binaries
 
 jobs:
+  rv32e-linux-x86-64:
+    name: rv32e Linux x86-64
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Checkout sources
+      uses: actions/checkout@v2
+      with:
+        submodules: recursive
+    - run: git clone --depth 1 --branch release/16.x https://github.com/llvm/llvm-project.git
```

Comment:
So this means we're using llvm15 and llvm16 in solang. How can we switch between them?

Reply:
Looks like we already switched, thanks!